### ISSUE TITLE
Pdf does not include svg-file

### ISSUE THREAD

**Issue by rob70** [incuna/django-wkhtmltopdf#139]
I have a template with an image of an empty form in the background, and divs positioned to fill out the empty blanks from context in the foreground.

The template is rendered by django as it should, and the pdf is produced, but without the image.

The html-code to include the image file is this: The property "data" is referencing the svg-file:

`div id="pg1" style="-webkit-user-select: none;"><object width="909" height="1286" data="/path/to/image.svg" type="image/svg+xml" id="pdf1" style="width:909px; height:1286px; background-color:white; -moz-transform:scale(1); z-index=0</object>`

Is there an obvious solution to this?

**Comment by johnraz** [comment]
This is mostly an issue with wkhtml itself and not this project. I'm gonna close unless you can find out that it works with wkhtml.

**Comment by rob70** [comment]
I will try to debug this if possible. 

I don't know if you are right about this being a wkhtmltopdf issue only, though. 

Just for the record, I can produce a pdf with the same picture through the command line that I cannot through django, which seems strange.

**Comment by johnraz** [comment]
I may be totally wrong about this. The best way to report this kind of bug is to submit a failing test case.

**Comment by rob70** [comment]
In a reply to my original question: The obvious place to look for missing image files is in the html file which is produced in the tmp directory. Look at the link to the image in the source code.

I found out that the link is the path set in the STATIC_ROOT-setting.

I read in an other post that STATIC_ROOT must be set, which is not the case in development.
Django looks for static files both in the app-directory and in cross-app STATICFILES_DIR directories.
STATIC_ROOT is a setting used for deployment and must be different from the other static directories, because it is used to collect static files from the other directories for convenience in deployment. (python manage.py collectstatic ) It is recommended that these files are served by a web server directly in production environment.

Perhaps this setting is used exclusively by the django-wkhtmltopdf script? If so, maybe this should be documented?

When I ensured that the image file was added to the folder referenced i STATIC_ROOT, it all worked out well.

However, I did find out that wkhtmltopdf indeed has a problem with svg-files. I had to convert it to a jpg for it to work.

Again, many thanks for this project.
